webservices/docs.py

```diff
@@ -267,13 +267,42 @@
 or `/totals` endpoints.
 
 When comparing the totals from reports to line items. the totals will not match unless you
-take out items where `"memoed_subtotal":true`. Memoed items are subtotals of receipts
-that are already accounted for in another schedule a line item.
+only look at items where `"is_individual":true` since the same transaction is in the data
+multipliable ways to explain the way it may move though different committees as an earmark.
```

Comment:
"multipliable" -> "multiple". Also, this sentence could be confusing--some wordsmithing and/or an example might help. Thoughts @emileighoutlaw?